studio/components/interfaces/Auth/SiteUrl/SiteUrl.tsx

```diff
@@ -4,31 +4,36 @@ import { useEffect } from 'react'
 import { Form, Input } from 'ui'
 import { boolean, number, object, string } from 'yup'
 
+import { useParams } from 'common'
 import {
   FormActions,
   FormHeader,
   FormPanel,
   FormSection,
   FormSectionContent,
 } from 'components/ui/Forms'
-import { useCheckPermissions, useSelectedProject, useStore } from 'hooks'
+import { useAuthConfigQuery } from 'data/auth/auth-config-query'
+import { useAuthConfigUpdateMutation } from 'data/auth/auth-config-update-mutation'
+import { useCheckPermissions, useStore } from 'hooks'
 
 const SiteUrl = observer(() => {
-  const { authConfig, ui } = useStore()
-  const { isLoaded } = authConfig
+  const { ui } = useStore()
+  const { ref: projectRef } = useParams()
+  const { data: config, isFetched: isLoaded } = useAuthConfigQuery({ projectRef })
```

Comment:
use isSuccess, add loading and error states

Reply:
Done.